initializr-service/src/main/resources/application.yml

```diff
@@ -1211,7 +1213,7 @@ initializr:
           groupId: org.springframework.statemachine
           artifactId: spring-statemachine-starter
           description: Build applications using state machine concepts
-          versionRange: 2.0.0.RC1
+          versionRange: 2.0.0.RELEASE
```

Comment:
Thanks for the PR. 

There is no need to remove that range since we've kept the history for now. I'll clean all the intermediate milestones once all projects have a proper 2.0 compatible GA. I'll revert that when polishing.